ops/check.sh

```diff
@@ -55,7 +55,9 @@ frontend_check(){
 }
 
 e2e_check(){
-  echo "e2e check"
+  cd frontend
+  pnpm install --no-frozen-lockfile
+  pnpm run e2e
```

Comment:
may be missing some steps for e2e check.
![image](https://github.com/au-heartbeat/Heartbeat/assets/33213627/4c5cf515-c5f2-4323-8338-f60f3c991390)

Reply:
get it